### PR TITLE
IRC kick handler

### PR DESCRIPTION
**Pull request by 10eMyrT**
This PR adds the `Kick Handler` to IRC. There were a lot of possible messages to send, by I have used `kickFmt = "* %s kicked %s from %s"` because I believe it is simple and clear. Your comments are welcomed.

### CODE REVIEW

File: internal/handlers/irc/handlers.go [hunk 1]
```diff
@@ -92,15 +93,30 @@ func quitHandler(c Client) func(*girc.Client, girc.Event) {
 	}
 }
 
+/*
+kickHandler handles the event when a user is kicked from the IRC channel.
+*/
+func kickHandler(c Client) func(*girc.Client, girc.Event) {
+	return func(gc *girc.Client, e girc.Event) {
+		c.logger.LogDebug("kickHandler triggered")
+		if c.TelegramSettings != nil && c.TelegramSettings.ShowKickMessage {
```
Comment (Zedjones) [inline internal/handlers/irc/handlers.go:102]: You can get rid of this check here
```go
if c.TelegramSettings != nil
```
It was unnecessarily added to the other handlers in a previous PR, I'm going to remove it from the other handlers in an upcoming PR

Comment (10eMyrT) [reply internal/handlers/irc/handlers.go]: Thanks for the review. I removed the check for `c.TelegramSettings != nil`.

**Comment by Zedjones** [comment]
Functionality looks good, do you think it might be good to include the reason for the kick in the message? Opinions? @jwflory @Tjzabel

**Comment by jwflory** [comment]
> Functionality looks good, do you think it might be good to include the reason for the kick in the message? Opinions? @jwflory @Tjzabel

I would like to include the kick reason in the message. This is how v1.x.x behaves I think.

**Comment by Tjzabel** [comment]
@10eMyrT since we merged in @Zedjones 's changes, this PR now has issues due to the `kickHandler` not being under the `ClientInterface`.

**Comment by 10eMyrT** [comment]
Just push a new commit to solve the problem.

**Comment by 10eMyrT** [comment]
It checks out now. We are good to go.

**Pull request merged**